From: SeongJae Park <sj@kernel.org>
Date: Sun, 22 Sep 2024 09:26:13 -0700
Subject: [PATCH] Docs/mm/damon: recommend academic papers top read and/or cite

Kernel documentation is the most up-to-date resource for DAMON that
supposed to be read by everyone who has interest in DAMON.  For some
cases, however, academic papers are better to be read and/or cited than
the kernel documentation because those have standardized format and good
citation systems.  Of course, only if the content of the paper is not
outdated as of the reads and cites.

DAMON has two academic papers[1,2], but some papers and articles are
still citing rendered kernel documentation web pags instead, maybe due
to the unclear guidance and the insufficient visibility.  Add a
paragraph for DAMON academic papers on kernel documentation.

[1] https://dl.acm.org/doi/abs/10.1145/3366626.3368125
[2] https://dl.acm.org/doi/abs/10.1145/3502181.353146

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/index.rst | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/mm/damon/index.rst b/Documentation/mm/damon/index.rst
index dafd6d028924..0cb11f3c7987 100644
--- a/Documentation/mm/damon/index.rst
+++ b/Documentation/mm/damon/index.rst
@@ -37,3 +37,10 @@ with no code but simple configurations.
 
 To utilize and control DAMON from the user-space, please refer to the
 administration :doc:`guide </admin-guide/mm/damon/index>`.
+
+There are two academic papers from `HPDC'22
+<https://dl.acm.org/doi/abs/10.1145/3502181.3531466>`_ and `Middleware19
+Industry <https://dl.acm.org/doi/abs/10.1145/3366626.3368125>`_ that introduces
+DAMON of Linux v5.16 and Linux v5.15.  If you prefer academic papers over
+kernel documentation and need to read and/or cite DAMON of the versions, please
+use those.  Later one (HPDC'22 paper) is more preferred in general.
-- 
2.39.5

